.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,21 @@
 {
   "name": "helium-csharp",
-  "image": "retaildevcrew/codespaces-dotnet",
```

Comment:
I'm assuming that without specifying the image or the build, it's using the upstream vscode image?

Reply:
correct. i also tested that docker commands still work with the upstream image without the "mount" option.